src/ParameterLists.f90

```diff
@@ -2739,33 +2776,28 @@ SUBROUTINE verifyTest_Paramtype(thisParam,reqParams,isMatch)
 !>        the structure AND values in two parameter lists.  If they are a match,
 !>        isMatch will be returned as true.  If not, false.  Assertion failures
 !>        will be printed for the parameter list values that fail.
-!> @param thisParam
-!> @param reqParams
-!> @param isMatch
+!> @param thisParam The parameter list on which to verify the values
+!> @param reqParams The reference parameter list and values
+!> @param e The exception handler to pass
+!> @param isMatch The logical if all parameter names and values are the same
 !>
     SUBROUTINE verifyList_Paramtype(thisParam,reqParams,e,isMatch)
       CLASS(ParamType),INTENT(INOUT) :: thisParam
       CLASS(ParamType),INTENT(IN) :: reqParams
-      CLASS(ExceptionHandlerType),INTENT(IN) :: e
+      CLASS(ExceptionHandlerType),INTENT(INOUT) :: e
```

Comment:
I'm bothered that the order of `e` and `isMatch` are reversed between `verifyList_Paramtype` and `validateReq_ParamType` seems like one more way the programmer can trip over their own feet.

Reply:
Done.  Good point.